src/ColoredConsole/ColoredConsole.csproj

```diff
@@ -1,70 +1,40 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>

```

Comment:
You should really target the lowest version of .NET Standard possible. As I said [previously](https://github.com/colored-console/colored-console/issues/62#issuecomment-240233328), I'd be surprised if this library could not target .NET Standard 1.0.

Reply:
I can't go back to 1.0 because `System.ConsoleColor` is only available in 1.3, but 1.3 runs fine. I will send an updated commit.

Reply:
I just pushed the updated commit. It now supports 1.3 which will demand .NET Framework 4.6 as per https://docs.microsoft.com/en-us/dotnet/standard/net-standard